theLudovyc/Building/Buildings.gd

```diff
@@ -1,27 +1,28 @@
 extends Object
 class_name Buildings
 
-enum Types { Placeholder, Warehouse, Residential, Producing }
+enum Types { Placeholder, Warehouse, Residential, Producing, Producing_Collecting }
 
 enum Ids { Warehouse, Tent, Lumberjack }
 
-enum Datas { Name, Type, Cost, Produce, Max_Workers }
+enum Datas { Name, Type, Cost, Collect, Produce, Max_Workers }
 
-const datas = {
+const datas: Dictionary = {
 	Ids.Warehouse: {Datas.Name: &"Warehouse", Datas.Type: Types.Warehouse},
 	Ids.Tent:
 	{
 		Datas.Name: &"Tent",
 		Datas.Type: Types.Residential,
-		Datas.Cost: [[Resources.Types.Wood, 1], [Resources.Types.Textile, 1]],
+		Datas.Cost: [[Resources.Types.Board, 1], [Resources.Types.Textile, 1]],
```

Comment:
If you look at the roadmap and code, I regroup resources in tiers.

T1 / Sailors => Gathering, Collecting, like get wood
T2 / Pioneers => First transform, like wood in planks
etc...

So, currently you work on the v0.02 and not on v0.01

Reply:
I thought this godot port should be a rework of the current game.

"Right now the main objective is to port over the [original project](https://github.com/unknown-horizons/unknown-horizons)'s existing feature set and assets and make a playable version out of it." 

In the current game the lumberjack collect wood and generates boards.
In oder to build a residental building you need boards and not wood.

Reply:
Added new building sawmill which uses wood to generate boards / planks.
Using the flipped image and icon of the lumberjack because no specific icon and image are in the old game :/